qutebrowser/browser/shared.py

```diff
@@ -23,6 +23,9 @@
 
 import jinja2
 
+import os
+import netrc
+
```

Comment:
Imports are [grouped like this](http://pep8.org/#imports):

```
import stlib
...

import lib
...

import qutebrowser.foo
...
```

so this shold be below `import html` above (without a blank line)